desktop/src/com/frostwire/gui/bittorrent/CreateTorrentDialog.java

```diff
@@ -99,6 +96,24 @@
     private String _invalidTrackerURL;
     private JButton _buttonClose;
 
+    private JComboBox _pieceSizeComboBox;
+    private JLabel _labelPieceSize;
+    private int pieceSize;
+    private static final String[] PIECE_SIZE_OPTIONS = {
```

Comment:
You can pass a `ComboBoxModel<E>`, which can return non-unidimensional objects.
https://docs.oracle.com/javase/7/docs/api/javax/swing/ComboBoxModel.html